PluginCore/PluginCore/Utilities/PathWalker.cs

```diff
@@ -98,8 +98,13 @@ private void ExploreFolderWithMasks(String path, String[] masks)
             foreach (String mask in masks)
             {
                 String[] files = Directory.GetFiles(path, mask);
+                String control = mask.Length == 5 && mask[0] == '*' && mask[1] == '.' ? mask.Substring(1) : null;
```

Comment:
Yeah it would fail with fancy matching after the initial `*.`. Not sure how to efficiently sove that, and if that's a use case we want to support at all.